config/environments/production.rb

```diff
@@ -74,28 +80,23 @@
   # Don't log any deprecations.
   config.active_support.report_deprecations = false
 
-  # Use default logging formatter so that PID and timestamp are not suppressed.
-  config.log_formatter = ::Logger::Formatter.new
-
-  # Use a different logger for distributed setups.
-  # require "syslog/logger"
-  # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
-
-  if ENV["RAILS_LOG_TO_STDOUT"].present?
-    logger           = ActiveSupport::Logger.new($stdout)
-    logger.formatter = config.log_formatter
-    config.logger    = ActiveSupport::TaggedLogging.new(logger)
-  end
-
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 
   config.action_mailer.delivery_method = :notify
   config.action_mailer.notify_settings = {
-    api_key: Rails.application.secrets.notify_api_key,
+    api_key: Rails.application.credentials.notify_api_key,
```

Comment:
You might find that this doesn't work and that you have to use ENV directly here, as I think this gets loaded before the initialisers. See https://github.com/alphagov/whitehall/blob/b0180e9fb01c7d5168047e457402bdbefb2bbc18/config/application.rb#L72

Reply:
ah right that makes sense. Thanks!